shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/compiler/PragmaNode.java

```diff
@@ -26,6 +29,43 @@
           }
           return result;
         });
+
+    DISPATCH.addKeyword(
+        "RequiredServices",
+        (input, output) -> {
+          final AtomicReference<List<String>> services =
+              new AtomicReference<>(Collections.emptyList());
```

Comment:
The empty list isn't necessary. If this doesn't get filled in, the code that uses it shouldn't be reached. No need to change it though.